lib/filters/global/actions.js

```diff
@@ -17,7 +17,12 @@ module.exports = {
                     body.format = 'json';
                     return restbase[req.method](req)
                     .then(function(res) {
-                        if (res.status !== 200) {
+                        if (
+                            res.status !== 200
```

Comment:
Is there a legitimate case where the body is empty, but the return status is 200? I'm actually not sure about res.body.query either.

For example, [this query](http://en.wikipedia.org/w/api.php?action=query&prop=revisions&meta=siteinfo&titles=Somepagethatdoesn%27texist%20Page&rvprop=user%7Ccomment&continue=) still sets `query` and `pages` despite not matching any page.